lib/mjml.rb

```diff
@@ -22,15 +22,15 @@ def self.check_version(bin)
   end
 
   def self.discover_mjml_bin
-    # Check for a global install of MJML binary
-    mjml_bin = 'mjml'
-    return mjml_bin if check_version(mjml_bin)
-
     # Check for a local install of MJML binary
     installer_path = (`npm bin` || `yarn bin`).chomp
     mjml_bin = File.join(installer_path, 'mjml')
     return mjml_bin if check_version(mjml_bin)
 
+    # Check for a global install of MJML binary
+    mjml_bin = 'mjml'
+    return `which #{mjml_bin}`.chomp if check_version(mjml_bin)
```

Comment:
What I find slightly surprising here is the fact that we check the version without prepending the path, but then return the absolute path. Did you consider passing the absolute path into `check_version`?